ParcelDistributionCenter/ParcelDistributionCenter.Web/Controllers/PackagesController.cs

```diff
@@ -61,6 +62,34 @@ public ActionResult DeletePackage(int packageNumber)
             TempData["MessageClass"] = "alert-danger";
             return RedirectToAction(nameof(DisplayPackages));
         }
+      
+        // GET: ProductsController/Delete/5
+        public ActionResult Delete(int id)
+        {
+            var model = _packageHandler.FindPackageByNumber(id);
```

Comment:
Tutaj musi być sprawdzenie, czy paczka nie jest nullem. Jeśli nie jest to widok a, jeśli jest to jakiś widok b.

Reply:
Poprawię to.